pkg/watch/module_release_meta_change.go

```diff
@@ -0,0 +1,174 @@
+package watch
```

Comment:
Also, why is the watch pkg in `pkg` not `internal`. I don't think this will be ever be used by any other modules so it should be kept in `internal, WDYT?

Reply:
Agree, with both of your points, will rename and move it!